packages/check/match.js

```diff
@@ -192,6 +194,7 @@ const stringForErrorMessage = (value, options = {}) => {
   return EJSON.stringify(value);
 };
 
+
```

Comment:
(side note) What's the deal with those empty lines here and in the next file? Am I missing something?